src/lib/holocene/navigation/full-nav.svelte

```diff
@@ -128,6 +128,14 @@
       </NavRow>
     {:then user}
       {#if user?.email}
+        <NavRow {isCloud} link={linkList.settings}>
```

Comment:
I wouldn't add this yet until we have a settings page in cloud. Also going to need a different guard for this since it's only for cloud.temporal.io

Reply:
Oh ya, good call 👍